docs/content/tutorial/update-cluster.md

```diff
@@ -17,7 +17,7 @@ The Postgres image is referenced using the `spec.image` and looks similar to the
 
 ```
 spec:
-  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.4-1
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-postgres:centos8-13.4-0
```

Comment:
shouldn't this be the param :question:

Reply:
If you read the paragraph it looks like this shouldn't change so i didn't change it to a param.  i didn't want to accidently update it as a mistake when it is specific to the versions

Reply:
sweet :+1: